src/Particle/Validator/Rule/Between.php

```diff
@@ -33,8 +33,8 @@ class Between extends Rule
      * @var array
      */
     protected $messageTemplates = [
-        self::TOO_BIG => '{{ name }} is too big, upper limit is {{ max }}',
-        self::TOO_SMALL => '{{ name }} is too small, lower limit is {{ min }}',
+        self::TOO_BIG => '{{ name }} must be less than {{ max }}',
+        self::TOO_SMALL => '{{ name }} must be greater than {{ min }}',
```

Comment:
mmmm, this was based upon your docs:
https://github.com/NickBelhomme/Validator/pull/1/files#diff-32dd1967ead18ebf2d79505c49c3ab09R99

but looking at your code, it is exclusive instead of inclusive.
https://github.com/NickBelhomme/Validator/pull/1/files#diff-27f522a63d2c64bf9c40275d522dd684R74